versions/1/manifest.spec.yml

```diff
@@ -106,6 +106,21 @@ spec:
         - "1.0.0-SNAPSHOT"
         - "1.0.0-dev.1"
   properties:
+    elasticsearch:
+      description: Elasticsearch requirements
+      type: object
+      additionalProperties: false
+      properties:
+        privileges:
+          description: Elasticsearch privilege requirements
+          type: object
+          additionalProperties: false
+          properties:
+            cluster:
+              description: Elasticsearch cluster privilege requirements
```

Comment:
Do you know if there is any doc/page describing all correct values for this property? You used `monitor/main`, but it might be a bit mysterious for people unfamiliar with this topic.

Do you think we should introduce regex pattern here?